trino/transaction.py

```diff
@@ -74,16 +72,14 @@ def begin(self):
                 self._id = response.headers[constants.HEADER_STARTED_TRANSACTION]
             status = self._request.process(response)
         self._request.transaction_id = self._id
-        logger.info("transaction started: " + self._id)
+        logger.info("transaction started: %s", self._id)
 
     def commit(self):
         query = trino.client.TrinoQuery(self._request, COMMIT)
         try:
             list(query.execute())
         except Exception as err:
-            raise trino.exceptions.DatabaseError(
-                "failed to commit transaction {}: {}".format(self._id, err)
-            )
+            raise trino.exceptions.DatabaseError(f"failed to commit transaction {self._id}") from err
```

Comment:
What does `from err` do? First time seeing this. Interesting.

Reply:
It's equal `throw new Exception("something went wrong", cause)` in Java.
You could see more info here [PEP-3134](https://www.python.org/dev/peps/pep-3134/)